api/opensource/mfx_dispatch/src/mfx_dispatcher.cpp

```diff
@@ -25,8 +25,13 @@
 #include <assert.h>
 
 #include <string.h>
+#if defined(_WIN32) || defined(_WIN64)
+    #include <windows.h>
+    #pragma warning(disable:4355)
```

Comment:
please, try to remove and try without it. If warning still persist - try to fix.